client/transport.c

```diff
@@ -103,141 +103,150 @@ static void *xdag_send_thread(void *arg)
 	return 0;
 }
 
-static int block_arrive_callback(void *packet, void *connection)
+static int process_recieved_block_nonce_type(struct xdag_block *recieved_block, void *connection)
```

Comment:
This function is used to process transport pseudo blocks which is used  for statistics and exchanging blocks and sums among nodes.

So better to use another name something like process_transport_block or others.

Reply:
Ok, I will rename this function and fix typos in the next PR.